packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -782,4 +784,55 @@ export class PhoenixObjects {
 
     return object;
   }
+
+  /**
+   * Type for drawing irregular calorimeter cell comprising 8 xyz vertices in arbitrary geometry.
+   * @param irrCells Properties of irregular calo cell.
+   * @param irrCells.layer Calorimeter layer
+   * @param irrCells.vtx Flattened list of 8 vertex coordinates (24 floats)
+   * @param irrCells.color [R,G,B] integer list
+   * @param irrCells.opacity value from 0 to 1
+   * @returns the cell
+   */
+  public static getIrregularCaloCell(irrCells: {
+    type: any;
+    layer: number;
+    vtx: any;
+    color: string;
+    opacity: any;
```

Comment:
Hi @sponce we looked together and are not fully sure how to address this. We checked that the browser runs for us (with and without IrregularCaloCells) before pushing, but somehow it's not working the LHCb ci. 
Should we just add the following line here?
`    uuid: string;`

Reply:
In the meantime, we've identified what might be the reason for different behavior in LHCb : we somehow use threejs 148 rather than 147. So I suppose moving back to 147 would make it work for us.

On the other hand, the code in indeed weird, as you set a member that does not exist. So indeed, I would add `uuid:string` or alternatively drop the line setting the uuid if it's not used. But I suppose it is in your case (for lhcb, I did that stop gap fix yesterday and it worked)